Assets/Scripts/Utilities/AudioZone.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AudioZone : MonoBehaviour 
+{
+	[SerializeField] AudioSource sourceData = null;
+	AudioSource playingClip = null;
+
+	void OnTriggerEnter( Collider col )
+	{
+		if ( col.GetComponent<PlayerActor>() )
+		{
+			playingClip = SoundManager.instance.Play3DSoundAndFollow( sourceData, transform );
+		}
+	}
+
+	void OnTriggerExit( Collider col )
```

Comment:
Please use a better variable name than `col`. It's not much harder to use a longer name since we have auto-complete, and it's good practice to use descriptive variable names.